<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterForm</class>
 <widget class="QWidget" name="RegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 173);</string>
  </property>
  <widget class="QLabel" name="usrname">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;用户名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="usr_let">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>270</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd_let_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>270</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd_let_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>270</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="password_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="password_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;确认密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>500</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QComboBox" name="question_cbx">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>您最喜欢的事情是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您母亲的姓名是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您父亲的姓名是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您配偶的姓名是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您的出生地是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您高中班主任的名字是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您初中班主任的名字是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您小学班主任的名字是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您的小学校名是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您的学号（或工号）是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您父亲的生日是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您母亲的生日是？</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>您配偶的生日是？</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="password_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;设置密保问题&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ans_let">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>270</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="questionAns">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;问题答案&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="usr_hint">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>140</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pwd_hint_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pwd_hint_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hint_lbl_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 224, 102);</string>
   </property>
   <property name="text">
    <string>密码长度为6-18位，由数字、字母和下划线组成</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hint_lbl_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 224, 102);</string>
   </property>
   <property name="text">
    <string>用户名长度为3-12个字符，由汉字、数字、字母和下划线组成</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ans_hint">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
